config/routes.rb

```diff
@@ -404,7 +404,7 @@
   get 'capture/index/:id.:format' => 'capture#index' 
 
   post 'comments/spectraset/:id.:format' => 'comments#spectraset' 
-  delete 'comments/delete/:id.:format' => 'comments#delete' 
+  post 'comments/delete/:id.:format' => 'comments#delete' 
```

Comment:
I think this is actually a delete route :thinking:

Reply:
Whoops my bad